docs/Manual_Install.md

```diff
@@ -71,9 +71,19 @@ On each machine create the file `/etc/clearwater/local_config` with the followin
     public_ip=<publicIP>
     public_hostname=<hostname>
     etcd_cluster="<comma separated list of private IPs>"
+    local_site_name=<name>
```

Comment:
I think it would be clearer the local|remote_site_name options were with the 'If you are creating ...' section below